AmplifyPlugins/Storage/Tests/StorageHostApp/StorageHostApp.xcodeproj/project.pbxproj

```diff
@@ -268,19 +286,17 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 684FB0AF28BEB07200C8A6EB /* Build configuration list for PBXNativeTarget "AWSS3StoragePluginIntegrationTests" */;
 			buildPhases = (
+				684FB0C428BEB81900C8A6EB /* Copy Configuration */,
 				684FB0A528BEB07200C8A6EB /* Sources */,
 				684FB0A628BEB07200C8A6EB /* Frameworks */,
 				684FB0A728BEB07200C8A6EB /* Resources */,
-				684FB0C428BEB81900C8A6EB /* Copy Configuration */,
```

Comment:
Moving from a post-build approach to pre-build. This way, we can rely on the **Copy Bundle Resources** build phase doing its job (instead of us doing it for it).